### ISSUE TITLE
Cloud databases

### ISSUE THREAD

**Issue by maxdzin** [Tahul/nuxt-edgedb#30]
Greetings!

Is it possible to make this module work with [EdgeDB cloud](https://cloud.edgedb.com)? If so, can you explain how exactly?

Thank you!

**Comment by Tahul** [comment]
Hello there :) 

It sure is, you can take advantage of [env variables](https://github.com/Tahul/nuxt-edgedb#production) to do so.

From EdgeDB Cloud, you should be provided with a DSN to use, you can use that in development, or use a local instance in develpment and use that Cloud DSN in production.

**Comment by maxdzin** [comment]
Hi!
Thank you for your response. I haven't had enough time yet to take a look at that, but will check it out soon.
Thank you!

**Comment by watzon** [comment]
EdgeDB Cloud doesn't supply DSN information. Instead they give you a secret key and expect you to use the CLI. I've been trying to debug why this doesn't work properly in Vercel and from what I can tell the CLI isn't being installed in the environment in Vercel so even though I'm providing the EDGEDB_INSTANCE and EDGEDB_SECRET_KEY env vars it still doesn't work.

Searching the code, the `installCli` configuration option doesn't appear to actually do anything?

**Comment by maxdzin** [comment]
Yes, I tried yesterday to use the EdgeDB created in Vercel and connect the application using the provided credentials, but no success so far.
Also, I haven't found any clear documentation on how to get this to work.

**Comment by watzon** [comment]
From what I can see from the code for this repo, it expects the edgedb cli to be available, and if it is it tries to make use of that. What I think it needs to do after that is check for the environment variables EDGEDB_INSTANCE and EDGEDB_SECRET_KEY and attempt to connect with the cloud that way, otherwise it should continue to fallback to the DSN.

The EdgeDB docs have Vercel working out of the box because the official EdgeDB JavaScript SDK [looks for those env vars](https://github.com/edgedb/edgedb-js/blob/3934b86f2309172b1720d9a59473086d6e94c1d5/packages/driver/src/conUtils.ts#L593) when it makes the connection. This just needs to do the same.
